### ISSUE TITLE
Yarn warning: "@sucrase/webpack-loader@1.0.2" has incorrect peer dependency "sucrase@^1.12.1"

### ISSUE THREAD

**Issue by iki** [alangpierce/sucrase#239]
yarn@1.7.0 reports the following warning when using latest sucrase@^2.2.0:
```
"@sucrase/webpack-loader@1.0.2" has incorrect peer dependency "sucrase@^1.12.1"
```

Maybe allowing both versions ^1.12.1 and ^2 in package.json would help. The range can be specified either as `^1.12.1 || ^2`, or as `1.12.1 - 2`, try it at the [semver check](https://jubianchi.github.io/semver-check/):
```
  "peerDependencies": {
    "sucrase": "1.12.1 - 2"
  },
  "devDependencies": {
    "@types/loader-utils": "^1.1.1",
    "sucrase": "1.12.1 - 2"
  },
```

**Comment by alangpierce** [comment]
Oops, thanks, you're right, I forgot to publish the integrations with updated peer dependencies when releasing sucrase 2.0. I'll fix for all of them.

**Comment by alangpierce** [comment]
I just released Sucrase 3.0 and updated all integrations to reference that, so all the peer deps should be in place now.
